pycbc/waveform/compress.py

```diff
@@ -479,14 +514,15 @@ def fd_decompress(amp, phase, sample_frequencies, out=None, df=None,
         hlen = len(out)
     if f_lower is None:
         imin = 0
-        f_lower = sample_frequencies[0]
+        f_lower = f_min
     else:
-        if f_lower >= sample_frequencies.max():
-            raise ValueError("f_lower is > than the maximum sample frequency")
-        imin = int(numpy.searchsorted(sample_frequencies, f_lower))
+        if f_lower >= f_max:
+            raise ValueError("f_lower is greater than the maximum "
+                             "sample frequency of the interpolants")
     start_index = int(numpy.floor(f_lower/df))
     # interpolate the amplitude and the phase
     if interpolation == "linear":
+        imin = int(numpy.searchsorted(sample_frequencies, f_lower))
```

Comment:
__[flake8]__
* [F841] local variable 'imin' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->